docs/setup/clouds/aws.md

```diff
@@ -0,0 +1,48 @@
+# AWS
+
+The runner have the capability to create resources on AWS. Follow the steps below to authenticate
+and configure this service
+
+## Credentials
+
+### Get credentials for Engineering-EC2User
+
+Create a .env file at the root of the project and put in it your AWS_ACCESS_KEY_ID, your AWS_SECRET_ACCESS_KEY and your AWS_SESSION_TOKEN.
+You will also have to fill in AWS_DEFAULT_REGION (us-west-2 for example) and GITHUB_TOKEN (your github token).
+
+## Config
+
+### Service
+
+Considering the application credentials has been setup on the service system,
+the remaining service configuration you'll need are:
+```yaml
+cloud_name: 'aws'
+cloud_config: {}
+```
```

Comment:
Took the liberty of applying suggestion some modifications, here's the main reasons of why:
* Removed engineering-ec2user as that's specific to Scality.
* Removed mention about `.env` because that's one method of doing it, we have multiple and the end goal is to have environment variables.
* Removed mention of GitHub token as it's defined in the setup guide. We should talk only about aws here
* Moving region from credentials to config as a region is not a credential
* AWS_SESSION_TOKEN is optional and not always required depending on your aws account settings.

```suggestion

## Credentials

Set the following environment variables, with your respective AWS credentials:

* `AWS_ACCESS_KEY_ID`
* `AWS_SECRET_ACCESS_KEY`
* `AWS_SESSION_TOKEN` (Optional)

## Config

### AWS Region

Configure the region where EC2 instances will be created with the `AWS_DEFAULT_REGION` environment variable. Per example `us-west-2`.

### Service

Considering the application credentials has been setup on the service system,
the remaining service configuration you'll need are:

```yaml
cloud_name: 'aws'
cloud_config: {}
```
```

Reply:
the suggestion is looking weird at the end, there's a conflict with the backquotes of the yaml, might need some manual edit

Reply:
Thanks for all of this, I'll change this thanks to your comments